x/simulation/simulate.go

```diff
@@ -252,11 +253,12 @@ type blockSimFn func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 
 // Returns a function to simulate blocks. Written like this to avoid constant
 // parameters being passed everytime, to minimize memory overhead.
-func createBlockSimulator(testingMode bool, tb testing.TB, w io.Writer, params Params,
```

Comment:
need a changelog for this

Reply:
I don't think we need a changelog for non exported functions.